addons/udes_get_info/models/models.py

```diff
@@ -65,16 +65,18 @@ def _get_info(self, level, info_fields, extra_fields):
         result = self[field_name]
         if isinstance(self[field_name], models.BaseModel):
             # Stop recursing once we reach level zero or if the field is empty.
-            if (level == 0) or (not result):
+            if level == 0:
                 continue
+            if not result:
+                _info = False
```

Comment:
Q: Are there impications to overwriting `_info` to be `False`, isn't that going to bubble up the recursion and blank out any other values which were previously there?

Reply:
ah, yes sorry, i didn't spot the `info[info_name] = _info` at the end, so it's only going to be changing that depths value to False and not affecting the rest of info. Gotcha